compiler/passes/errorHandling.cpp

```diff
@@ -937,6 +943,24 @@ bool ErrorCheckingVisitor::enterCallExpr(CallExpr* node) {
       bool inThrowingFunction = false;
       if (FnSymbol* parentFn = toFnSymbol(node->parentSymbol)) {
         inThrowingFunction = parentFn->throwsError();
+
+        if (!inThrowingFunction && isTaskFun(calledFn)) {
+          taskFunctionDepth = 1;
```

Comment:
Could this increment and decrement taskFunctionDepth? I'm concerned that we'll get something wrong with nested task functions otherwise.